app/assets/javascripts/pages.js

```diff
@@ -1,6 +1,6 @@
 //data page
 
-$(document).ready(function() {
+if($("#data-dump").length) {
```

Comment:
I'm not familiar with the codebase, but shouldn't this remain `document.ready` and maybe get this extra check as an additional condition within the ready callback function?